idact/detail/add_cluster_app/actions_parser.py

```diff
@@ -0,0 +1,8 @@
+def parse_actions(file_name: str):
+    """ Parser for the actions file.
+    The file should contain bash commands
+    line after line.
+    """
+    with open(file_name) as file:
+        list_of_actions = file.read().split('\n')
```

Comment:
why not `readlines()`

Reply:
Because it reads lines with the trailing '/n' character. 
I used the solution from stack [https://stackoverflow.com/questions/12330522/reading-a-file-without-newlines]